README.md

```diff
@@ -183,6 +201,28 @@ ruff check [--fix]
 You can install a Ruff VSCode plugin to check errors and format files from
 the editor.
 
+## Run huggingface transformer models
+
+For contributors to torchprime, `tp run` also supports running the huggingface
+trainer, for debugging and comparison.
+
+To run the huggingface trainer, you can clone
+[huggingface/transformers][hf-transformers] under
+torchprime and name it as `local_transformers`. This allows you to pick any
+branch or make code modifications in transformers for experiment:
+
+```sh
+git clone https://github.com/huggingface/transformers.git local_transformers
+```
+
+If huggingface transformer doesn't exist, torchprime will automatically clone
+the repo and build the docker for experiment. To switch to huggingface models,
+add flag `--use-hf` to `tp run` command:
+
+```sh
+tp run --use-hf torchprime/hf_models/train.py
+```
+
```

Comment:
Would it be cleaner to add this to `hf_models/README.md` and mention `torchprime/hf_models` in structure section? Just hoping to keep the main README.md show high level information

Reply:
I like this idea. It's also in line with confusing newcomers less.